README.md

```diff
@@ -1,5 +1,130 @@
+
+Based on this, our project generated the following generalized defintion for pseudonymization:
+
+> the processing of personal information in a manner that renders the personal information no longer attributable to a specific consumer without the use of additional information.
+
+#### De-Identification
+
+The term de-identification is used within US-based cybersecurity and data privacy concepts. The National Institutes of Standards and Technology ("NIST") relies on this concept for removing identifiers from personal information, defining de-identification as:
+
+> general term for any process of removing the association between a set of identifying data and the data subject [2].
+
+By removing this association between the identifier and the information, information becomes de-identified, thereby reducing certian privacy concerns. Additional uses of the term "de-idenfitication" can be found in the Health Insurance Portability and Accountability Act (HIPAA) and the California Consumer Privacy Act of 2018. 
+
+Based on these sources, our project generated the following generalized definition for "de-identification":
+
+> Information that cannot reasonably identify, relate to, describe, be capable of being associated with, or be linked, directly or indirectly, to a particular individual.
+
+#### Anonymization
+
+The concept of annonymization is used across a variety of different legal frameworks and regions. For example, NIST defines this term as:
+
+> the process that removes the association between the identifying dataset and the data subject [2]. 
+
+Further, while the term annonymization is not defined within the GDPR, it is used by the European Data Protection Board ("EDPB") in further explaining the impact of the GDPR within the context of geolocation tracking. [3] The Guidelines state that
+
+> Anonymisation refers to the use of a set of techniques in order to remove the ability to link the data with an identified or identifiable natural person against any “reasonable” effort. This “reasonability test” must take into account both objective aspects (time, technical means) and contextual elements that may vary case by case (rarity of a phenomenon including population density, nature and volume of data). If the data fails to pass this test, then it has not been anonymised and therefore remains in the scope of the GDPR. ¶ 15
+
+Futher, these Guidelines outline three factors to determine the "robustmenss of anonymization": 
+
+> (i) singling-out (isolating an individual in a larger group based on the data); (ii) linkability (linking together two records concerning the same individual); and (iii) inference (deducing, with significant probability, unknown information about an individual). ¶ 16
+
+Based on this guidance, our project generated the following generalized defintion for "anonymization":
+
+> A process that removes the association between the identifying dataset and the data subject.
+
```

Comment:
is this stronger than de-identification? i get a little confused on the inference factor. So, if we could infer someone's home zip code based on GPS data, then it wouldn't be anonymized, i think. So, the standard here is very high and perhaps impossible to anonymize unless we use differential privacy?
Differential privacy is basically "I receive no ADDED information on person X by having this dataset"